backend/capellacollab/sessions/hooks/session_token.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+# SPDX-License-Identifier: Apache-2.0
+
+
+import datetime
+
+from capellacollab.permissions import models as permissions_models
+from capellacollab.users.tokens import crud as tokens_crud
+
+from . import interface
+
+
+class SessionTokenIntegration(interface.HookRegistration):
+    """Create a PAT valid for the duration of the session."""
+
+    def configuration_hook(
+        self,
+        request: interface.ConfigurationHookRequest,
+    ) -> interface.ConfigurationHookResult:
+        token, password = tokens_crud.create_token(
+            db=request.db,
+            scope=permissions_models.GlobalScopes(
+                user=permissions_models.UserScopes(
+                    sessions={permissions_models.UserTokenVerb.GET}
+                )
+            ),
+            title=f"Session {request.session_id}",
+            user=request.user,
+            description=(
+                "This PAT is managed by the Collaboration Manager."
+                " It will be revoked when the session is terminated"
+                " and is only valid for the duration of the session."
+                " Manual deletion may lead to unexpected behavior."
+            ),
+            expiration_date=datetime.date.today()
+            + datetime.timedelta(
+                days=2
+            ),  # Maximum duration is until end of the next day.
+            source="session automation",
+        )
+
+        return interface.ConfigurationHookResult(
+            environment={"CAPELLACOLLAB_SESSION_TOKEN": password},
+            config={"session_token_id": token.id},
+        )
```

Comment:
As far as I can tell, CAPELLACOLLAB_SESSION_TOKEN is used for tools like guacamole to identify users (because we couldn't do authentication via the API). Would it make sense to use a different name here? At least we should update the docs with the new usage.

Reply:
Ohhh, that's a really good point. This might actually lead to unwanted behaviour. I'll rename it to `CAPELLACOLLAB_SESSION_API_TOKEN`.